project3.qmd

```diff
@@ -17,28 +17,59 @@ editor_options:
   chunk_output_type: console
 ---
 
----
+------------------------------------------------------------------------
+
+## Introduction
+
+Beginning in 2011, Stack Overflow has conducted an annual survey for developers to participate in. This survey provides insights on a wide variety of topics, such as employment status, annual salary, language knowledge, location, and much more.
+
+As aspiring data scientists, we aim to answer the following questions:
+
+-   What are the most popular programming languages?
+
+-   What are the highest paying programming languages?
+
+-   Do certain developer types make more money than others?
+
+-   Do different regions make more money than others?
+
+-   Does age have an impact on salary?
+
+-   Are people more likely to be employed full-time if they have a higher level degree?
+
+-   Is there a gender wage gap in the industry?
+
+-   Do larger companies hire more experienced people?
+
+-   Does knowing more than one language increase your salary?
+
+-   Does having a higher level degree impact your salary?
+
+In this project, we will use the Stack Overflow survey data to answer the above questions, and hopefully provide more insight to our fellow aspiring data scientists.
+
+------------------------------------------------------------------------
 
 #### Project overview
+
 ![](./images/overview.svg)
 
----
+------------------------------------------------------------------------
 
 #### Data engineering overview
 
 Our data engineering pipeline does three things.
 
-1. For each year, download raw survey data from our s3 cache. Unify and coalesce columns on each year, and then union together.    
-2. Explode wide certain multi-value columns. For example, "PlatformWorkedWith" contains both AWS and Google Cloud.
-3. Save the CSV file in the root directory so the markdown can load it from cache.
+1.  For each year, download raw survey data from our s3 cache. Unify and coalesce columns on each year, and then union together.\
+2.  Explode wide certain multi-value columns. For example, "PlatformWorkedWith" contains both AWS and Google Cloud.
+3.  Save the CSV file in the root directory so the markdown can load it from cache.
 
 [funcitons.r](https://github.com/tonythor/krijudato/blob/develop/functions.r) contains our data engineering pipeline code.
 
 Also, to prepare for this project, we had to do a column analysis. If you'd like to see what went into that, review and run our [admin.r](https://github.com/tonythor/krijudato/blob/develop/admin.r) file.
 
----
+------------------------------------------------------------------------
```

Comment:
I'll move this through, but in readme, you only need three hyphens together to create a horizontal rule.